demo/examples/src/wallet_connect.cc

```diff
@@ -136,22 +136,15 @@ int main(int argc, char *argv[]) {
             std::cout << "signature length=" << sig1.size() << std::endl;
         } else {
             WalletConnectTxEip155 info;
-            info.to = rust::String(
-                std::string("0x") +
-                address_to_hex_string(result.addresses[0].address).c_str());
-            info.common.gas_limit = "21000"; // gas limit
-            info.common.gas_price = "10000"; // gas price
-            info.value = "100000000000000";  // 0.0001 eth
-            info.data = rust::Vec<uint8_t>();
-            info.common.nonce = "1";
-            info.common.chainid = 777;
-            rust::Vec<uint8_t> signedtx =
-                client->sign_eip155_transaction_blocking(
+            info.to = "0xA914161b1b8d9dbC9c5310Fc7EBee5A5B18044b7";
```

Comment:
this address ok?
how about read from env ?

Reply:
ok

Reply:
adding a env is tedious (on small examples), just convert back to send CRO to the connected wallet address itself.